src/components/project-pages/EroticStories.tsx

```diff
@@ -1,33 +1,128 @@
 import React from "react";
 import styled from "styled-components";
-import Timer from "../Timer";
-import { FlexboxProps, flexbox, layout, LayoutProps } from "styled-system";
-import statue from "../assets/project-page/statue.png";
+import { FlexboxProps, flexbox, layout, LayoutProps, grid, GridProps, space, SpaceProps, typography, TypographyProps } from "styled-system";
+import EroticStoriesPreview from "./EroticStoriesPreview";
+import alasImg from "./../assets/erotic-stories/alas-bateando.jpg";
+import ombligosImg from "./../assets/erotic-stories/ombligos.jpg";
+import { useTranslation } from "react-i18next";
 
-const Main = styled.main<FlexboxProps>`
-  display: flex;
-  height: 100vh;
-  overflow: hidden;
-  flex-direction: column;
-  ${flexbox}
+
+const Main = styled.main<TypographyProps>`
+  ${typography};
+  height: 100%;
 `;
 
-const Container = styled.div<FlexboxProps>`
-  display: flex;
+const Grid = styled.div<GridProps & FlexboxProps & LayoutProps>`
+  ${grid};
   ${flexbox};
+  ${layout};
 `;
 
-const Img = styled.img<LayoutProps>`
-  ${layout};
+const H1 = styled.h1<TypographyProps & SpaceProps>`
+  text-transform: uppercase;
+  ${space};
+  ${typography};
+`;
+
+const H2 = styled.h1<TypographyProps & SpaceProps>`
+  ${space};
+  ${typography};
+`;
+
+
+const EroticImageContainer = styled.div<GridProps & SpaceProps>`
+  ${grid};
+  ${space};
+  object-fit: contain;
+  height: fit-content;
+`;
+
+const EroticImage = styled.img<SpaceProps>`
+height: 100%;
+width: 100%;
+${space};
+`;
+
+const ScrollableTextContainer = styled.div<GridProps & SpaceProps>`
+  ${grid};
+  ${space};
+  object-fit: contain;
+  overflow: scroll;
+`;
+
+
+const ScrollableText = styled.p`
+  overflow: scroll;
 `;
 
+
+interface EroticStoryProps {
+  title: string;
+  img: string;
+  text: string;
+  alt: string;
+}
+
+
+
+
 const EroticStories = () => {
+  const { t } = useTranslation();
+  const eroticStories: EroticStoryProps[] = [
+    {
+      title: "Ombligos",
+      img: ombligosImg,
+      text: t("erotic-stories.poem"),
+      alt: "Artwork image",
+    },
+    {
+      title: "Alas Bateando",
+      img: ombligosImg,
+      text: t("erotic-stories.poem"),
+      alt: "Artwork image",
+    }
+  ];
+
+const eroticStory1 = eroticStories[0]
+const eroticStory2 = eroticStories[1]
+
   return (
-    <Main justifyContent="center" alignItems="center">
-      <Container justifyContent="center" alignItems="center">
-        <Img src={statue} alt="statue icon" maxWidth="30%" />
-      </Container>
-      <Timer endDate="2020-11-20" />
+    <Main fontFamily="secondary" textAlign="justify">
+      <H1 fontSize={5} mb={4}>
+       Erotic Stories
+      </H1>
+      <H2 fontSize={5} mb={4}>
+        my wings are beating
+      </H2>
+      <Grid display={["flex", "flex", "flex", "grid"]}
```

Comment:
i think using Grid here adds more code than if it was just `display: flex`, i'll push up a different version of this file and you can compare

Reply:
or actually, ive just been through the Flex flow and think its cleaner to have Grid all the way through, rather than Flex

Reply:
This is great! I especially like this version on smaller desktop screens, I think it looks clean too.

Only problem we have is we've lost the scroll on mobile. I've just had a play with `overflow`,  `height`  and `min-height` but so far no luck. Any other ideas?

Reply:
im working on this...